src/sentry/tasks/post_process.py

```diff
@@ -624,15 +624,22 @@ def process_code_mappings(job: PostProcessJob) -> None:
 
         org_slug = project.organization.slug
         next_time = timezone.now() + timedelta(hours=1)
-        if features.has("organizations:derive-code-mappings", event.project.organization):
+        has_normal_run_flag = features.has(
+            "organizations:derive-code-mappings", event.project.organization
+        )
+        has_dry_run_flag = features.has(
+            "organizations:derive-code-mappings-dry-run", event.project.organization
+        )
+
+        # Right now EA orgs have both flags on, thus, only supporting dry run
+        if has_normal_run_flag and event.data["platform"] == "python":
```

Comment:
This is what prevents JS events from being processed for realz. When we're ready we will remove this.